mix.exs

```diff
@@ -5,7 +5,7 @@ defmodule Chroner.MixProject do
     [
       app: :chroner,
       version: "0.1.0",
-      elixir: "~> 1.8.0",
+      elixir: "~> 1.8",
```

Comment:
Why not 1.10?

Reply:
@sasa1977 I wanted to define minimal working requirement for this project. Not to push developers to update their elixir version to latest release in order to run some dependency if not necessary.

Reply:
This is primary built for vbt projects, right? If so, I suggest sticking to the latest version. Even if you aim to reach a wider population, I'd still keep it at the latest version. Keeping backward compatibility is time-consuming (you need to run CI tests for all supported versions), while upgrading to latest Elixir is generally not all that difficult.